ubuntudesign/vanilla-framework
# 496 code block update

## Done

* updated all code variants
* drive-by: updated color documentation
* drive-by: moved reset up in the main vanilla.scss

## QA

1. go to vf.demo.haus and see that all the code resembles the images
 * &lt;code> element - /base/code/ - [inline](https://github.com/ubuntudesign/vanilla-design/blob/master/Code/%3Ccode%3E.png)
 * &lt;pre>&lt;code> element  - /base/code/ - [dark code block](https://github.com/ubuntudesign/vanilla-design/blob/master/Code/Code%20block.png)
 * code snippets- /patterns/code-snippet/ - [code snippet dark](https://github.com/ubuntudesign/vanilla-design/blob/master/Code/Code%20snippet%20dark.png) and [code snippet]https://github.com/ubuntudesign/vanilla-design/blob/master/Code/Code%20snippet%20light.png
 * numbered code blocks - /patterns/code-numbered/ - [light](https://github.com/ubuntudesign/vanilla-design/blob/master/Code/Code%20block%20numbered.png)

## Details

Fixes #496
